driver-core/src/main/com/mongodb/client/model/expressions/ArrayExpression.java

```diff
@@ -61,4 +63,29 @@
      */
     T reduce(T initialValue, BinaryOperator<T> in);
 
+    IntegerExpression size();
+
+    T arrayElemAt(IntegerExpression i);
+
+    default T arrayElemAt(final int i) {
+        return this.arrayElemAt(of(i));
+    }
+
+    T first();
+
+    T last();
+
+    BooleanExpression contains(T contains);
+
+    ArrayExpression<T> concatArrays(ArrayExpression<T> array);
```

Comment:
Did we decide to prefer closer match to expression name (in this case `$concatArrays`)when it's easy to do so?  `concat` makes more sense in context of `ArrayExpression` type.

Reply:
Agreed.